plugins/optimization-detective/optimization.php

```diff
@@ -295,9 +218,9 @@ function od_optimize_template_output_buffer( string $buffer ): string {
 		if (
 			$style
 			&&
-			preg_match( '/background(-image)?\s*:[^;]*?url\(\s*[\'"]?(?<background_image>.+?)[\'"]?\s*\)/', (string) $style, $matches )
+			preg_match( '/background(-image)?\s*:[^;]*?url\(\s*[\'"]?\s*(?<background_image>.+?)\s*[\'"]?\s*\)/', (string) $style, $matches )
 			&&
-			! str_starts_with( $matches['background_image'], 'data:' )
+			! $is_data_url( $matches['background_image'] )
```

Comment:
Just a little hardening bugfix to account for whitespace around the URL or, more unlikely, the scheme to be `DATA:` which _does_ work in browsers.